includes/collection/class-actors.php

```diff
@@ -203,6 +204,14 @@ public static function get_by_resource( $uri ) {
 			// Check for http(s) URIs.
 			case 'http':
 			case 'https':
+				// Check locally stored remote Actor.
+				$post = self::get_remote_by_uri( $uri, true );
```

Comment:
```suggestion
				$post = self::get_remote_by_uri( $uri );
```
Maybe `get_remote_by_uri()` used to accept a second arg? Looks like it no longer does.